docs/documentation/platform/event-subscriptions.mdx

```diff
@@ -0,0 +1,147 @@
+
+Event messages are structured according to the [CloudEvents specification](https://github.com/cloudevents/spec), providing a standardized format for cross-service communication. For client-side delivery, the system employs Server-Sent Events (SSE) streams, which offer efficient unidirectional communication without requiring specialized client libraries. This lightweight approach ensures seamless real-time updates while maintaining broad compatibility across different client environments.
+
+<Note>
+ Event Subscriptions are designed for real-time communication and do not include persistence or replay capabilities—events are delivered once and are not stored for future retrieval.
+</Note>
+
+## Supported Resources
+
+You can currently subscribe to notifications for the following resources and event types:
+
+- **Secrets**
+ - `secret:created`: Triggered when a secret is created
+ - `secret:updated`: Triggered when a secret is updated
+ - `secret:deleted`: Triggered when a secret is deleted
+
+## Permissions Setup
+
+To receive events on a supported resource, the identity must have `Subscribe` action permission on that resource.
+
+Follow these steps to set up the necessary permissions:
+
+<Steps>
+ <Step title="Select a project and copy the Project ID">
+   ![Select Project](/images/platform/events/select-project.png)
+   
+   On your project page, open **Project Settings** from the sidebar.
+   
+   In the Project name section, click **Copy Project ID** to copy your Project ID, or extract it from the URL:
+   `https://app.infisical.com/project/<your_project_id>/settings`
+ </Step>
+
+ <Step title="Navigate to Access Management and open Project Roles">
+   ![Project Detail](/images/platform/events/project-detail.png)
+   ![Project Access](/images/platform/events/project-access.png)
+   
+   Navigate to **Access Management**, then select **Project Roles**.
+ </Step>
+
+ <Step title="Select an existing role or create a new one">
+   ![Project Role](/images/platform/events/project-role.png)
+   
+   You can either edit an existing role or create a new role for event subscriptions.
+ </Step>
+
+ <Step title="Assign policies to the role">
+   ![Role Detail](/images/platform/events/role-detail.png)
+   
+   Select the specific resources that the role should have access to.
+   
+   ![Add policy](/images/platform/events/add-policy.png)
+ </Step>
+
+ <Step title="Enable the Subscribe action in permissions">
+   ![Policy setting](/images/platform/events/policy-setting.png)
+   
+   Ensure the **Subscribe** action is selected for the relevant resources and events.
+   
+   <AccordionGroup>
+     <Accordion title="Add conditions to your permissions">
```

Comment:
May be as an accordion can you provide some examples like full acess, some scoped access images